### DISCUSSION TITLE
Create / remove routes in production ( runtime ). Hot swap routes

### DISCUSSION THREAD

**Discussion by odranoelBR** [nuxt/nuxt#24007] [Ideas]
I have search a lot about this feature because in e-commerce apps, there always a needing of include new routes withtou release / build.
We can't just update the vue router instance running in any client browser.
Someone know a way to do it ?

**Comment by viii47** [comment]
I don't think that this is possible within the framework of vue3/nuxt itself. Seems to me you need a different approach to inject new routes on-the-fly. If it's content that you update regularly, then you could make dynamic route outlets `/content/[...name].vue/ etc.` which load specific content from a backend source -> which in term checks if that content you require is available or not.
